src/modules/sdlog2/sdlog2.c

```diff
@@ -103,6 +104,8 @@
 #include "sdlog2_format.h"
 #include "sdlog2_messages.h"
 
+#define PX4_EPOCH_SECS 1234567890ULL
```

Comment:
@LorenzMeier Can you add a similar define to mavlink_receiver for SYSTEM_TIME message please.